buildpack.toml

```diff
@@ -30,6 +30,14 @@ dir = "vendor/ruby/heroku-20"
 url = "https://heroku-buildpack-ruby.s3.us-east-1.amazonaws.com/heroku-22/ruby-3.1.4.tgz"
 dir = "vendor/ruby/heroku-22"
 
+[[publish.Vendor]]
+url = "https://heroku-buildpack-ruby.s3.us-east-1.amazonaws.com/heroku-22/amd64/ruby-3.1.4.tgz"
+dir = "vendor/ruby/amd64/heroku-24"
+
+[[publish.Vendor]]
+url = "https://heroku-buildpack-ruby.s3.us-east-1.amazonaws.com/heroku-22/arm64/ruby-3.1.4.tgz"
+dir = "vendor/ruby/arm64/heroku-24"
```

Comment:
I think we should only vendor the AMD64 Ruby binary in the classic buildpack registry release, since otherwise Codon is paying the boot time delay price for every build (Ruby or not) for an architecture that will never get used on Heroku.

Even if the ARM64 binary isn't included, that arch will still work on eg Dokku, since it will be dynamically downloaded by the bootstrap script.

Reply:
Thanks for the review, I removed it. Can you take another look?